src/postgres/src/test/regress/sql/yb_pg_triggers.sql

```diff
@@ -1316,12 +1314,17 @@ create table trigpart1 partition of trigpart for values from (0) to (1000);
 create trigger trg1 after insert on trigpart for each row execute procedure trigger_nothing();
 create table trigpart2 partition of trigpart for values from (1000) to (2000);
 create table trigpart3 (like trigpart);
-alter table trigpart attach partition trigpart3 for values from (2000) to (3000);
 select tgrelid::regclass, tgname, tgfoid::regproc from pg_trigger
   where tgrelid::regclass::text like 'trigpart%' order by tgrelid::regclass::text;
+-- Disabling the below tests temporarily because each run of the test can
+-- throw one of two errors. The error message displayed contains the first dependency
+-- resulting from the scan of pg_depend preventing the delete of the trigger. The trigger
+-- depends on the partition table itself and on the trigger created on the partitioned
+-- table, and either of the two error messages are possible.
+/*
```

Comment:
For this cases, instead of disabling them, we temporarily change verbosity level for these commands to omit the detail lines.
For instance:
```
\set verbosity terse
DROP .. 
\set verbosity default
```
Alternatively, you can use the native SQL way of doing `SET client_min_messages TO 'WARNING'` and then `RESET client_min_messages`.
The latter is probably cleaner if it works because it's just native SQL (rather than a `ysqlsh` command) but (at least) one of them typically works to stabilize test output in these cases.

Reply:
The problem here was that was differing error messages... And there doesn't seem to be a way to silence Error messages as far as I could google :( Is there anything else I can do?